### DISCUSSION TITLE
Does xagg support longitude 0-360 degrees?

### DISCUSSION THREAD

**Discussion by helsharif** [ks905383/xagg#54] [Q&A] [answered]
Hello, 
I was wondering if xagg supports the use of both -180 to +180 and 0 to 360 degrees longitude conventions?
I have some netcdf files for example that use one or the other longitude conventions and want to make sure that xagg works properly with both. 
Also, does the polygon used with xagg have to use the same longitude convention as the data being aggregated?
Thank you

**Comment by ks905383** [comment, marked as answer]
> I was wondering if xagg supports the use of both -180 to +180 and 0 to 360 degrees longitude conventions?

Yes it does! (0:360 files are internally converted to -180:180 when you run the code). 

> Also, does the polygon used with xagg have to use the same longitude convention as the data being aggregated?

Nope - as long as the `gdf` has a valid crs. Under the hood, the crs is converted to EPSG:6931-6933 (depending on the domain) before processing to match the cartesian setup from above.
